.github/ISSUE_TEMPLATE.md

```diff
@@ -1,35 +1,41 @@
-Make sure you are running the latest version of Home Assistant before reporting an issue.
+## First
+- [ ] I have read and understand the [Contributing](https://github.com/home-assistant/home-assistant/CONTRIBUTING.md) documentation.
 
-You should only file an issue if you found a bug. Feature and enhancement requests should go in [the Feature Requests section](https://community.home-assistant.io/c/feature-requests) of our community forum:
+## Description
+<!--- Describe your changes. -->
 
-**Home Assistant release (`hass --version`):**
+## Platform Details
+Component/Platform: 
+'hass --version': 
+'python --version': 
 
+## Pull Requests
```

Comment:
There is no need to mention pull requests at all for an issue template. There is no code included here.